src/libdrakvuf/win-processes.c

```diff
@@ -1202,6 +1269,317 @@ bool win_find_mmvad(drakvuf_t drakvuf, addr_t eprocess, addr_t vaddr, mmvad_info
     return false;
 }
 
+static bool win_traverse_mmvad_node(drakvuf_t drakvuf, addr_t node_addr, mmvad_callback callback, void* callback_data)
+{
+    int depth = 0;
+    bool is_win7 = vmi_get_winver( drakvuf->vmi ) <= VMI_OS_WINDOWS_7;
```

Comment:
Why `<=`?

Reply:
This was copied from `win_find_mmvad`. I think the author of the code (@sasza8 in commit 74530d) could clarify the meaning of the comparison.